Solution1/DataAccessLayer/Repositories/WeatherRepository.cs

```diff
@@ -0,0 +1,24 @@
+﻿using DataAccessLayer.Configuration;
+using DataAccessLayer.Models;
+using DataAccessLayer.Repositories.Abstract;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace DataAccessLayer.Repositories
+{
+    public class WeatherRepository : IWeatherRepository
+    {
+        readonly RepositoryContext _repository;
+
+        public WeatherRepository(RepositoryContext repository)
+        {
+            _repository = repository;
+        }
+        
+        public async Task BulkSaveWeatherAsync(IEnumerable<Weather> weather)
```

Comment:
name at list weatherList, it's not clear from the name that it's not one weather

Reply:
+